objects.py

```diff
@@ -0,0 +1,44 @@
+import turtle
+
+win = turtle.Screen()
+win.title("Game")
+win.bgcolor("black")
+win.setup(width=800, height=600)
+win.tracer(0)
+
+# rockets
+rocket_left = turtle.Turtle()
+rocket_left.speed(0)
+rocket_left.shape("square")
+rocket_left.color("green")
+rocket_left.shapesize(stretch_len=1, stretch_wid=5)
+rocket_left.penup()
+rocket_left.goto(-350, 0)
+
+rocket_right = turtle.Turtle()
+rocket_right.speed(0)
+rocket_right.shape("square")
+rocket_right.color("red")
+rocket_right.shapesize(stretch_len=1, stretch_wid=5)
+rocket_right.penup()
+rocket_right.goto(350, 0)
```

Comment:
This code pretty much duplicates the above block. Therefore, it make sense to create left and write rocket on `create_rocket` function call.